fmf/base.py

```diff
@@ -320,7 +339,7 @@ def update(self, data):
         log.debug("Data for '{0}' updated.".format(self))
         log.data(pretty(self.data))
 
-    def adjust(self, context, key='adjust', undecided='skip'):
+    def adjust(self, context, key='adjust', undecided='skip', on_rule=None):
```

Comment:
Because of line 439 on which `adjust()` is called on a child structure, and the original `on_rule` is passed to it. Probably not necessary, I'll refactor this part a bit.

Reply:
Refactored in c096e5f